modules/exploits/windows/local/cve_2020_1054_drawiconex_lpe.rb

```diff
@@ -67,25 +68,6 @@ def initialize(info = {})
           'DisclosureDate' => '2020-02-20'
         )
```

Comment:
Given the source code limits the buffer to a max size of 0x1000 we should probably also update this to limit the payload size to the same amount.

Reply:
Fixed with https://github.com/rapid7/metasploit-framework/pull/15384/commits/a92468c635a72a9f624fa3216364025d4a8936cf